paddle/fluid/framework/details/op_handle_base.cc

```diff
@@ -187,6 +187,15 @@ void OpHandleBase::RunAndRecordEvent(const std::function<void()> &callback) {
     std::function<void()> method = callback;
     for (auto &p : dev_ctxes_) {
       method = [method, p, this]() {
+        VLOG(10) << "cudadevicecontext:"
+                 << static_cast<platform::CUDADeviceContext *>(p.second)
+                 << ", dev_id:"
+                 << boost::get<platform::CUDAPlace>(p.first).device;
+
+        VLOG(10) << "events:"
+                 << &events_.at(
+                        boost::get<platform::CUDAPlace>(p.first).device);
```

Comment:
Why do you add those code here?  Is it just used to debug for your program?

Reply:
Ye. Will clean it!

Reply:
Done.